jwst/ami/ami_analyze.py

```diff
@@ -54,9 +54,9 @@ def apply_LG_plus(input_model, filter_model, oversample, rotation,
         ysize = 80
         xstop = xstart + xsize - 1
         ystop = ystart + ysize - 1
-        input_model.data = input_model.data[ystart-1:ystop, xstart-1:xstop]
-        input_model.dq = input_model.dq[ystart-1:ystop, xstart-1:xstop]
-        input_model.err = input_model.err[ystart-1:ystop, xstart-1:xstop]
+        input_model.data = input_model.data[ystart - 1:ystop, xstart - 1:xstop]
```

Comment:
This is actually one place where my personal preference is to disagree with codestyle, in that putting in white space then "attaches" the "1" visually with the 2nd argument of the slice, rather than keeping it with the first. But I'm willing to forgo personal preference in this case.

Reply:
Because we are specifically ignoring E231 (enforce whitespace after `,` and `:`), I did not bother to run `autopep8` with this code to do any corrections. We could turn it on and do the checks.  This would then enforce that the following:

```python
input_model.dq = input_model.dq[ystart-1:ystop, xstart-1:xstop]
```

becomes

```python
input_model.data = input_model.data[ystart - 1: ystop, xstart - 1: xstop]
```

This is where going part way, encourages one to go all the way.  Thoughts?  I find either of these more readable than what I corrected it to, but we now don't allow the former because we want spaces between arithmatic operators for this PR.  Hmm.

Reply:
Personally I think the first example above is fine. The 2nd ends up with everything so spread out that it actually makes it harder to follow (IMHO). I think space before and after arithmetic operators should only be enforced when used as part of an actual statement that's computing something, but not when they're used with arguments to a function or slicing.

By the way, is there an "Oxford comma" option for the checker to apply?  I'm a BIG believer in that one.

Reply:
I agree.